docs/zh_CN/plugins/sinks/influx.md

```diff
@@ -39,7 +39,7 @@
 ```json
 {
   "id": "influx",
-  "sql": "SELECT * from  demo_stream where temperature < 50",
+  "sql": "SELECT * from  demo_stream where temperature > 50",
```

Comment:
Could you please explain why making this change?

Reply:
You can check the conditions in the example usage, the configuration file (influxRule.txt) is "an example for selecting a temperature greater than 50 degrees", so I change it

Reply:
OK，thank you. Could you please also change the same file in the en_US? Thanks

Reply:
Ok, I will modify it